plugins/destination/elasticsearch/client/client_test.go

```diff
@@ -22,5 +31,8 @@ func TestPlugin(t *testing.T) {
 		destination.PluginTestSuiteTests{
 			SkipMigrateOverwriteForce: true,
 			SkipMigrateAppendForce:    true,
+
```

Comment:
Once we add force migrate mode we can remove the `SkipMigrateOverwriteForce` and `SkipMigrateAppendForce`, but this is out of scope of this PR